stack2nix/Stack2nix/Stack.hs

```diff
@@ -134,6 +135,7 @@ data Location
 -- Parsers for package indices
 sha256Suffix :: ReadP r Sha256
 sha256Suffix = string "@sha256:" *> many1 (satisfy (`elem` (['0'..'9']++['a'..'z']++['A'..'Z'])))
+                                 <* optional (char ',' <* many1 (satisfy isDigit))
```

Comment:
Can we add a comment here, that we optionally parse a file size?

Reply:
Added comment in b03eb7e